src/CookieStore.ts

```diff
@@ -12,6 +12,7 @@ interface ResponseLike {
 }
 type Store = Map<string, StoreEntry>
 type StoreEntry = Map<string, Cookie>
+type StringifiedCookie = Omit<Cookie, 'expires'> & { expires?: string };
```

Comment:
Suggestion: what do you think if we name it `CookieString`?

Reply:
Yes, that sounds more in line with the other types. I will change it.